rotkehlchen/db/dbhandler.py

```diff
@@ -3521,3 +3524,45 @@ def get_history_events(self, filter_query: HistoryEventFilterQuery) -> List[Hist
                     f'Failed to read history event from database. {str(e)}',
                 )
         return result
+
+    def rows_missing_prices_in_base_entries(
+        self,
+        filter_query: HistoryEventFilterQuery,
+        ignored_ids: Set[str],
+    ) -> List[Tuple[str, FVal, Asset, Timestamp]]:
+        """
+        Get missing prices for history base entries based on filter query
+        """
+        query, bindings = filter_query.prepare()
+        query = 'SELECT identifier, amount, asset, timestamp FROM history_events ' + query
+        cursor = self.conn.cursor()
+        result = []
+        for identifier, amount_raw, asset_name, timestamp in cursor.execute(query, bindings):
+            if identifier in ignored_ids:
+                continue
```

Comment:
Few things off here.

1. Do not do in cursor.execute(). Separate concerns please.
2. Why do the identifier check outside of sqlite? Keep that filter you had in the previous partof the PR. Do not pass ignored_ids here but just pass the proper filter including ignored ids.
3. 
```suggestion
        query = 'SELECT identifier, amount, asset, timestamp FROM history_events ' + query
        result = []
        cursor = self.conn.cursor()
        cursor.execute(query, bindings)
        for identifier, amount_raw, asset_name, timestamp in cursor:
```